packages/config/src/load.js

```diff
@@ -52,9 +53,8 @@ export async function loadNuxtConfig ({
   if (configFile) {
     // Clear cache
     clearRequireCache(configFile)
-
-    options = esm(module)(configFile) || {}
-
+    const requirModule = createRequire(module)
```

Comment:
```suggestion
    const requireModule = createRequire(module)
```

Reply:
@Atinux PR is still WIP pushing fixes in few minutes :)